plugin-flex-ts-template-v2/src/feature-library/contact-history/flex-hooks/components/SideNav.tsx

```diff
@@ -0,0 +1,16 @@
+import * as Flex from '@twilio/flex-ui';
+
+import { FlexComponent } from '../../../../types/feature-loader';
+import { isFeatureEnabled } from '../../config';
+import ContactHistorySideLink from './ContactHistorySideLink';
+
+export const componentName = FlexComponent.SideNav;
+export const componentHook = function addContactHistoryToSideNav(flex: typeof Flex) {
+  if (!isFeatureEnabled()) {
+    return;
+  }
+
+  flex.SideNav.Content.add(<ContactHistorySideLink viewName="contact-history" key="contact-history-side-nav" />, {
+    sortOrder: 1,
```

Comment:
Does this appear in the wrong spot when running Hosted Flex? The admin plugin adds itself as 0.